File: code-samples/java/consignment-stock.java
HttpRequest request = HttpRequest.newBuilder()
    .uri(URI.create("https://vendor-api-staging.jumia.com/consignment-stock?businessClientCode=string&sku=string"))
    .header("Authorization", "Bearer ")
    .method("GET", HttpRequest.BodyPublishers.noBody())
    .build();
HttpResponse<String> response = HttpClient.newHttpClient().send(request, HttpResponse.BodyHandlers.ofString());
System.out.println(response.body());

File: code-samples/python/consignment-stock.py
import requests

url = "https://vendor-api-staging.jumia.com/consignment-stock"

querystring = {"businessClientCode":"string","sku":"string"}

payload = ""
headers = {"Authorization": "Bearer "}

response = requests.request("GET", url, data=payload, headers=headers, params=querystring)

print(response.text)